src/validation_workflow/deb/validation_deb.py

```diff
@@ -0,0 +1,60 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import time
+
+from system.execute import execute
+from test_workflow.integ_test.utils import get_password
+from validation_workflow.api_test_cases import ApiTestCases
+from validation_workflow.download_utils import DownloadUtils
+from validation_workflow.validation import Validation
+from validation_workflow.validation_args import ValidationArgs
+
+
+class ValidateDeb(Validation, DownloadUtils):
+    def __init__(self, args: ValidationArgs) -> None:
+        super().__init__(args)
+
+    def installation(self) -> bool:
+        try:
+            for project in self.args.projects:
+                set_password = f' env OPENSEARCH_INITIAL_ADMIN_PASSWORD={get_password(str(self.args.version))}' if project == "opensearch" else ""
+                execute(f'sudo dpkg --purge {project}', ".")
+                execute(f'sudo {set_password} dpkg -i {os.path.basename(self.args.file_path.get(project))}', str(self.tmp_dir.path))
+                time.sleep(80)
+        except:
+            raise Exception("Failed to install Opensearch")
+        return True
+
+    def start_cluster(self) -> bool:
+        try:
+            for project in self.args.projects:
+                execute(f'sudo systemctl enable {project}', ".")
+                execute(f'sudo systemctl start {project}', ".")
+                time.sleep(20)
```

Comment:
Why is there a need to sleep 20 since opensearch systemd is using notify type and will wait until success.
Even if you have to wait why not just do a check of API calls every 5sec instead of blindly waiting for 20sec?
Different server spec can change the delay of this startup.

Reply:
Implemented this for all the distributions